pom.xml

```diff
@@ -751,7 +753,41 @@
                         <groupId>${jdbc.groupId}</groupId>
                         <artifactId>${jdbc.artifactId}</artifactId>
                         <version>${jdbc.version}</version>
-                        <scope>compile</scope>
+                    </dependency>
+                    <dependency>
+                        <groupId>org.hibernate</groupId>
+                        <artifactId>hibernate-entitymanager</artifactId>
+                        <version>4.1.7.Final</version>
+                        <exclusions>
+                            <exclusion>
+                                <groupId>cglib</groupId>
+                                <artifactId>cglib</artifactId>
+                            </exclusion>
+                            <exclusion>
+                                <groupId>commons-logging</groupId>
+                                <artifactId>commons-logging</artifactId>
+                            </exclusion>
+                        </exclusions>
+                    </dependency>
+                    <dependency>
+                        <groupId>org.hibernate</groupId>
+                        <artifactId>hibernate-core</artifactId>
+                        <version>4.1.7.Final</version>
```

Comment:
Good question. There is at least one class that has moved to a different package that the plugin uses. It is incompatible with newer versions of Hibernate.